lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -1871,129 +1921,52 @@ module.exports = (config) => {
       }
       debug('in fetchData', data);
 
-      const settings = {
-        units: {
-          bg: 'mg/dL', // Even though the pump can be in mmol/L, we read the mg/dL settings values
-          carb: 'grams',
-        },
-      };
+      debug('Read pump history');
+      const threeMonthsAgo = new Date(new Date().valueOf() - 7884e6);
+      // const threeMonthsAgo = new Date(new Date().valueOf() - 1210e6);
```

Comment:
This can be removed?

Reply:
Yeah, I'll remove it after I've finished all my testing.